lib/Utility/SuiteLogger.php

```diff
@@ -83,19 +83,19 @@ public function log($level, $message, array $context = array())
                 break;
             case LogLevel::WARNING:
```

Comment:
why everything is fatal here?

Reply:
This is for testing purposes mostly. Also the existing logger does not yet support the new levels.

Reply:
I have added some methods which let me temporarily set the log level.